experimental/packages/opentelemetry-sdk-node/src/utils.ts

```diff
@@ -88,3 +88,7 @@ export function getResourceDetectorsFromEnv(): Array<DetectorSync> {
     return resourceDetector || [];
   });
 }
+
+export function filterBlanksAndNulls(list: string[]): string[] {
+  return list.map(item => item.trim()).filter(s => s !== 'null' && s !== '');
```

Comment:
Is there any reason not to check for other stringified invalid values here, like `'undefined'`?

Reply:
idk just moved it here don't really want to change it